README.md

```diff
@@ -120,6 +117,45 @@ curl localhost:8080
 # Output: Hello World from PHP HTTP function!
 ```
 
+# Run your function in Cloud Run
+
+First you must have the [gcloud SDK][gcloud] installed and [authenticated][gcloud-auth].
+
+Additionally, you need to have a Google Cloud project ID for the
+[Google Cloud Project][gcp-project] you want to use.
+
+Next, build the container using the example `Dockerfile`:
+
+```sh
+docker build . \
+    -f vendor/google/cloud-functions-framework/examples/hello/Dockerfile \
```

Comment:
I think this needs to be:
```
docker build examples/hello
```

Reply:
This path assumes the library is installed as a vendored dependency. Although even then, it would be cleaner as 

```
docker build vendor/google/cloud-functions-framework/examples/hello \
    -t my-cloud-function
```

Reply:
Ah, I just cloned this repo.

Reply:
Oh, also, if you do that it uses the `index.php` in the `examples/hello` repo, but if you follow the steps in the README, it expects you to create a custom `index.php` and use that.

The example can be used both ways. We could potentially outline BOTH in the readme (cloning the repo and build the example AND create your own and use composer). But it may be confusing to include both. WDYT?

Reply:
Ah, I see.  Yeah, both approaches have value.  I tend to create examples that are self-contained and runnable without modification.

Reply:
I cleaned up the steps so it's clearer that you are deploying one from an application directory.

Reply:
I think it's the age-old tradeoff between ensuring the user does something and understands what they're doing, and getting something to work quickly. If they go in and run the example, they'll probably complete the steps faster but not have as strong of a grasp as to why. I think what we have here is a good compromise.